governance/polls/templates/Generic/$GenericApprovalWithoutMajority - $date_MONTH_DD,_YYYY.md

```diff
@@ -0,0 +1,57 @@
+---
+title: $GenericChange - $date_MONTH_DD,_YYYY
+summary: Signal your support or opposition to $GenericChange.
+discussion_link: $discussion_link
+parameters:
+  input_format:
+    type: choose-free
+    options: [1,2]
+    abstain: [0]
+  victory_conditions:
+    - { type : approval }
+  result_display: approval-breakdown
+options:
+   0: Abstain
+   1: $Option1
+   2: $Option2
+start_date: $YYYY-MM-DDT16:00:00
+end_date: $YYYY-MM-DDT16:00:00
+---
+# Poll:$GenericChange - $date_MONTH_DD,_YYYY
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the $poll_source. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on $date_DAY,_MONTH_DD at 16:00 UTC.
+
+**This is an approval vote.**
+- **You may vote for multiple options.**
+- **You should select all the options that you would support in an executive vote.**
+- **Unselected options signal that you do not approve of those options and would not support the option(s) in an executive vote.**
+- **If you wish to abstain, that should be your one and only choice.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes:
+* $GenericChangeA
+* $GenericChangeB
+
+Please review the discussion [thread]($discussion_link) to help inform your position before voting.
+
+## Outcomes
+
+**The non-'Abstain' option with the most votes is deemed the winner and the following actions will be taken:**
+* This change will be included in an upcoming Executive Vote as the Protocol Engineering Core Unit's schedule allows.
+* It is expected that this Executive Vote will take place within 30 days of this poll passing, absent external factors.
+* If the Executive Vote passes, then these changes will become active in the Maker Protocol after the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) has expired.
+
+**If the votes for the 'No' option equal or exceed the votes for the 'Yes' option then no further action will be taken at this time.**
```

Comment:
The assumption with this poll type is that there isn't a 'no' and a 'yes' option and that it's more of a choice between two options and that status-quo isn't available. So, given that, this doesn't make the most sense.

Reply:
fixed